exercises/concept/lucas-numbers/.docs/instructions.md

```diff
@@ -0,0 +1,38 @@
+# Instructions
+
+You are a huge fan of the [Numberphile Youtube channel](https://www.youtube.com/watch?v=PeUbRXnbmms) and you just saw a cool video about the _Lucas Number Sequence_. You want to create this sequence using elixir.
```

Comment:
elixir-lang.org always capitalizes "Elixir" everywhere and I think we should too
```suggestion
You are a huge fan of the [Numberphile Youtube channel](https://www.youtube.com/watch?v=PeUbRXnbmms) and you just saw a cool video about the _Lucas Number Sequence_. You want to create this sequence using Elixir.
```